Resources/Textures/Objects/Misc/ParcelWrap/wrapped_parcel.rsi/meta.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": 1,
+  "license": " CC BY-SA 4.0",
+  "copyright": "wrapped_parcel © 2025 by Centronias (https://github.com/centronias) is licensed under CC BY 4.0",
```

Comment:
I made the sprites myself, so attribution's to just me. If somebody can come up with a way to adds some attribution to the `box.rsi` `box` sprite's creator, I'd be happy to do that, but I'm not sure who did that and the [attribution in that file is, uh, unintelligible](https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Objects/Storage/boxes.rsi/meta.json#L4) to me.